codecov.yml

```diff
@@ -3,6 +3,4 @@ codecov:
 
 coverage:
   status:
-    patch:
-      default:
-        threshold: 1%
+    patch: off
```

Comment:
why are we disabling this?
does this not disable codecov for all the PRs? 
is this what you want?

Reply:
The message was `@codecov codecov/patch — 70.00% of diff hit (target 91.36%)` with a failed status. According to the [docs](https://docs.codecov.io/docs/commit-status#patch-status), the 70% comes from the coverage of the diff and it was required to be at least as high as the project coverage of 91.36%, which is already very high. I don't know if we want to enforce such a strict coverage requirement on all PRs.

Reply:
I'm fine with removing the patch codecov 👍

Reply:
Aha! 
Thanks for the clarification Bogdan!